### PR TITLE
Add a stage to saucelabs session for integration test

### PR DESCRIPTION
**Pull request by katayal**
Adding a stage to saucelabs session name

### CODE REVIEW

File: integration/js/utils/WebderiverSauceLabs.js [hunk 1]
```diff
@@ -91,9 +91,7 @@ const getSauceLabsConfig = (capabilities) => {
     extendedDebugging: true,
     capturePerformance: true,
     crmuxdriverVersion: 'beta',
-    tunnelIdentifier: process.env.TRAVIS_JOB_NUMBER,
-    commandTimeout: 600,
-    idleTimeout: 1000
+    tunnelIdentifier: process.env.TRAVIS_JOB_NUMBER
```
Comment (aburnson) [inline integration/js/utils/WebderiverSauceLabs.js]: Looks like this filename has a typo in it? `s/WebderiverSauceLabs.js/WebdriverSauceLabs.js/`

Comment (katayal) [reply integration/js/utils/WebderiverSauceLabs.js]: Oh yes will fix it

File: CHANGELOG.md [hunk 2]
```diff
@@ -89,5 +89,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix unhandled Promise rejection in DefaultScreenSharingSession#start
 - Fix canary deployment script
 - Fix sip call integration test
+- Add stage to saucelabs session name for integration tests
```
Comment (simmkyu) [inline CHANGELOG.md]: This section is for 1.1.0. Please add it to the "Unreleased" section (line 17).

File: integration/js/utils/SdkBaseTest.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ class SdkBaseTest extends KiteBaseTest {
     this.url = this.url + '?m=' + kiteConfig.uuid;
     this.meetingTitle = kiteConfig.uuid;
     this.testName = testName;
-    this.capabilities["name"] = `${testName}-${process.env.TEST_TYPE}`;
+    this.capabilities["name"] = process.env.STAGE != undefined ? `${testName}-${process.env.TEST_TYPE}-${process.env.STAGE}`: `${testName}-${process.env.TEST_TYPE}`;
```
Comment (simmkyu) [inline integration/js/utils/SdkBaseTest.js]: nit: Use strict inequality (```!==```).

File: CHANGELOG.md (hunk 2 above)
Comment (katayal) [reply CHANGELOG.md]: oh yes will fix it

File: integration/js/utils/SdkBaseTest.js (hunk 3 above)
Comment (katayal) [reply integration/js/utils/SdkBaseTest.js]: yes will fix it

**Pull request merged**